vision/unit_tests/test_client.py

```diff
@@ -81,6 +81,51 @@ def test_image_with_client(self):
         image = client.image(source_uri=IMAGE_SOURCE)
         self.assertIsInstance(image, Image)
 
+    def test_multiple_detection_from_content(self):
+        from google.cloud.vision.feature import Feature
+        from google.cloud.vision.feature import FeatureTypes
+        from unit_tests._fixtures import LABEL_DETECTION_RESPONSE
+        from unit_tests._fixtures import LOGO_DETECTION_RESPONSE
+        RETURNED = LABEL_DETECTION_RESPONSE
+        LOGOS = LOGO_DETECTION_RESPONSE['responses'][0]['logoAnnotations']
+        RETURNED['responses'][0]['logoAnnotations'] = LOGOS
```

Comment:
This will mutate the resource for future users of `_fixtures`. You don't want your tests to have side effects, so you should copy when putting it into `returned`?